.circleci/config.yml

```diff
@@ -10,6 +10,12 @@ parameters:
   releases-slack-channel:
     type: string
     default: dps-releases
+  test-executor-packages:
+    type: string
+    default: "activities adjudications alerts appointments courtsentencing csip incidents locations sentencing visits"
+  test-executors-count:
+    type: integer
+    default: 11
```

Comment:
List all packages in the test project that will run on a parallel executor. Any other packages found will be grouped toegether on another executor.

Note that the executor count must be 1 more than the list entries, and is declared here to hopefully make that obvious. There is validation to check the value is correct.